test/extensions/filters/http/admission_control/config_test.cc

```diff
@@ -0,0 +1,114 @@
+#include <chrono>
+
+#include "envoy/extensions/filters/http/admission_control/v3alpha/admission_control.pb.h"
+#include "envoy/extensions/filters/http/admission_control/v3alpha/admission_control.pb.validate.h"
+
+#include "common/stats/isolated_store_impl.h"
+
+#include "extensions/filters/http/admission_control/admission_control.h"
+#include "extensions/filters/http/admission_control/evaluators/default_evaluator.h"
+
+#include "test/mocks/runtime/mocks.h"
+#include "test/mocks/server/mocks.h"
+#include "test/mocks/thread_local/mocks.h"
+#include "test/test_common/simulated_time_system.h"
+#include "test/test_common/utility.h"
+
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+
+using testing::NiceMock;
+using testing::Return;
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace AdmissionControl {
+namespace {
+
+class AdmissionControlConfigTest : public testing::Test {
```

Comment:
you'll want tests that verify what happens when invalid input is given to the filter (either here or in the evaluator test)

Reply:
I'm checking for invalid codes. What else were you thinking?

Reply:
I was thinking about the RELEASE_ASSERTS that I suggested you change to exceptions